src/models.py

```
@@ -0,0 +1,102 @@
+"""
+競馬データベースモデル定義
+
+このモジュールは競馬データを格納するためのSQLAlchemyモデルを定義します。
+主要なエンティティ：Race（レース情報）とResult（レース結果）
+"""
+
+from datetime import date
+from typing import List, Optional
+from sqlalchemy import Column, Integer, String, Date, Float, ForeignKey, Text
```

Ruff (0.11.9)

`sqlalchemy.Text` imported but unused

Remove unused import: `sqlalchemy.Text`

(F401)